lib/tests/cleanup_before.rb

```diff
@@ -28,7 +28,11 @@ def run!(args:)
           elsif OS.mac?
             if Dir.glob("#{HOMEBREW_CELLAR}/*").present?
               # moving is much faster than deleting.
-              test "bash", "-c", "mv #{HOMEBREW_CELLAR}/* /tmp"
+              tmp_dir = "/tmp/homebrew-cellar"
+              # tmp_dir shouldn't exist but try deleting it just in case
+              test "bash", "-c", "rm -rf #{tmp_dir}"
```

Comment:
This would remove any speed benefits from moving because we're deleting and moving.